src/apis/myPage.ts

```diff
@@ -0,0 +1,31 @@
+import { instance } from '@/apis/axios';
+
+export const requestUserInfo = async (): Promise<{ nickname: string }> => {
+  const res = await instance.get<{ data: { content: { nickname: string } } }>('/api/users/me');
```

Comment:
```suggestion
interface UserType { data: { content: { nickname: string } } }

  const res = await instance.get<UserType>('/api/users/me');
```
type은 분리해서 관리하는편이 가독성에도 도움이됩니다. data라는 key값이 중볻된 코드가 많던데, 하나의 공통 타입을 만들어서 상속시키눈 형태로 사용해 보는건 어떨까요? 유지보수에도 도움이 된답니다!

Reply:
프로젝트 초기에는 응답 구조가 일관되지 않았기 때문에 개별로 작성하고 있었는데요, 말씀주신 것처럼 이번 PR에서 작업한 부분은 겹치는 부분이 많아서 ApiResponse<T>처럼 공통타입 만들어서 써볼수 있을 것 같습니다. 리뷰 작성해주셔서 감사합니다!

Reply:
@zzzRYT 제가 리뷰어 선택을 안 눌렀었네요😱 항상 리뷰 남겨주시는데 제가 뭔가 잘못 눌렀나봐요... 죄송합니다😢 그리고 리뷰 달아주셔서 감사합니다!